SourceryFramework/Sources/Parsing/FileParser.swift

```diff
@@ -56,6 +56,7 @@ public final class FileParser {
     fileprivate var annotations: AnnotationsParser!
     fileprivate var inlineRanges: [String: NSRange]!
     fileprivate var inlineIndentations: [String: String]!
+    fileprivate var forceParse:[String] = ["AutoCoding"]
```

Comment:
This and initializer default value should be removed IMO

Reply:
Without a default value, all tests would need to be changed.

Reply:
Default Value removed